README.md

```diff
@@ -5,13 +5,15 @@ Setting up your environment
 ---------------------------
 
 1. Install nodejs
-1. Run ```npm install```
+2. Run `npm install`
+3. Run `bower install`
+4. Run `gulp watch` and edit scss files.
```

Comment:
Same for gulp:
```
npm install -g gulp
```
Furthermore, we can add these libraries as dependencies to ./package.json